test/containsAdjunctTest.html

```diff
@@ -103,11 +129,31 @@
             }
         ]
     }
-
 console.log('[[a] b [c]]')
 console.log('expected: false')
 console.log('actual: ' + containsAdjunct(z))
 
+//automated test
+describe('[a [b [c]]]', function() {
+            it('containsAdjunct(x)', function() {
```

Comment:
Restate these to say "does not contain an adjunct" if the expected value is false, or "contains an adjunct" if the expected value is true.